### PR TITLE
Add UserInput-UserOutput workflow

### PR DESCRIPTION
**Pull request by Kerono4ka**
<ol>
  <li>Create TWO new components: UserInput and UserOutput</li>
  <li>UserInput should hold an input element, UserOutput two paragraphs</li>
  <li>Output multiple UserOutput components in the App component (any paragraph texts of your choice)</li>
  <li>Pass a username (of your choice) to UserOutput via props and display it there</li>
  <li>Add state to the App component (=> the username) and pass the username to the UserOutput component</li>
  <li>Add a method to manipulate the state (=> an event-handler method)</li>
  <li>Pass the event-handler method reference to the UserInput component and bind it to the input-change event</li>
  <li>Ensure that the new input entered by the user overwrites the old username passed to UserOutput</li>
  <li>Add two-way-binding to your input (in UserInput) to also display the starting username</li>
  <li>Add styling of your choice to your components/ elements in the components - both with inline styles and stylesheets</li>
</ol>

form

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
 import './App.css';
+import UserInput from './UserInput/UserInput';
+import UserOutput from './UserOuput/UserOutput';
+
+class App extends React.Component {
+  state = {
+    username: 'Olyaaaa',
+  }
+
+  userInputHandler = (event) => {
```
Comment (karpolan) [inline src/App.js:11]: Use `handleAbcQwerty()` as name for event handlers. It is not obligated, but a standard

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
+    if (event.key === "Enter") {
+      this.setState({
+        username: event.target.value,
+      })
+    }
+  }
+
+  render() {
+    const size = 2;
+    const outputElements = Array(size).fill( <UserOutput username={this.state.username}/>);
```
Comment (karpolan) [inline src/App.js:21]: Wow, nice "clonning" patern :)

Comment (karpolan) [inline src/App.js:21]: Ususally all states and props are **destructed** in the beginning of `render()` function

```
render() {
  const { className} = this.props
  const { username } = this.state
...
```

**Review by karpolan — APPROVED** [review]
Actually great :)

File: src/App.js (hunk 1 above)
Comment (Kerono4ka) [reply src/App.js:11]: I'll remember, thank you!

File: src/App.js (hunk 2 above)
Comment (Kerono4ka) [reply src/App.js:21]: Oo, I get it

**Pull request merged**